app/time_tracking/event/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Event(models.Model):
+    title       = models.CharField(max_length=120)
+    description = models.TextField(blank=True, null=True)
+    start_date  = models.DateField(auto_now_add=False)
```

Comment:
no need to specify `auto_now_add=False`, as it will be `False` by default

Reply:
The redundance `auto_now_add=False` is now removed.